README.md

```diff
@@ -423,11 +426,11 @@ Choose one or the other.
 
 ### Using Docker (Linux Only)
 
-The command shown here starts the `humble` version of the `micro-ros-agent` Docker image.
+The command shown here starts the `jazzy` version of the `micro-ros-agent` Docker image.
 However, always make sure to use a version of the Agent image which corresponds to the version of ROS 2 that is being used.
 
-With ROS 2 Foxy, use `microros/micro-ros-agent:foxy`.
 With ROS 2 Humble, use `microros/micro-ros-agent:humble`.
```

Comment:
```suggestion
With ROS 2 Foxy, use `microros/micro-ros-agent:foxy`.
With ROS 2 Humble, use `microros/micro-ros-agent:humble`.
```

Regardless of the deprecation of the Foxy version, let's just be clear about which version of the image should be used.

We only have one version of the documentation right now, so we can't remove this yet I believe.